node_modules/oae-transfer/lib/rest.js

```diff
@@ -0,0 +1,124 @@
+/*!
+ * Copyright 2017 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var OAE = require('oae-util/lib/oae');
+var TransferAPI = require('oae-transfer');
+
+/**
+ * @REST postTransferCreate
+ *
+ * Create a new transfer
+ *
+ * @Server      tenant
+ * @Method      POST
+ * @Path        /transfer/create
+ * @FormParam   {Context}           ctx                 Context
+ * @FormParam   {string}            emailOrigin         The email
+ * @FormParam   {string}            emailTarget         The new email       		
+ * @FormParam   {string}            idUserOrigin        The ID user       		
+ * @Return      {transfer}                              The created transfer
+ * @HttpResponse                    200                 Transfer created
+ * @HttpResponse                    400                 EmailTarget cannot be empty
+ * @HttpResponse                    400                 Transfer not found
+ */
+OAE.tenantRouter.on('post', '/api/transfer/create', function (req, res) {
+    TransferAPI.createTransfer(req.ctx, req.body.emailOrigin, req.body.emailTarget, req.body.idUserOrigin, function (err, transfer) {
+        if (err) {
+            return res.send(err.code, err.msg);
+        }
+        return res.send(200, transfer);
+    });
+});
+
+/**
+ * @REST getTransferByEmail
+ *
+ * Get transfer
+ *
+ * @Server      tenant
+ * @Method      GET
+ * @Path        /transfer/getTransferByEmail/:email
+ * @FormParam   {Context}           ctx                 Context
+ * @FormParam   {string}            email               Email               
+ * @Return      {transfer}                              The created transfer
+ * @HttpResponse                    200                 Transfer created
+ * @HttpResponse                    400                 Transfer not found
+ */
+OAE.tenantRouter.on('get', '/api/transfer/getTransferByEmail/:email', function (req, res) {
```

Comment:
The note I just wrote on `3akai-ux` [PR#4124](https://github.com/oaeproject/3akai-ux/pull/4124) applies here as well.